pkg/mixer/lint.go

```diff
@@ -204,3 +221,21 @@ func lintGrafanaDashboards(filename string, vm *jsonnet.VM, errsOut chan<- error
 		}
 	}
 }
+
+func isLintExcluded(ruleName string, alertName string, cf *lint.ConfigurationFile) bool {
+	exclusions, ok := cf.Exclusions[ruleName]
+	excluded := false
```

Comment:
Nit; this could be eliminated by simply returning true anywhere that there is assignment, and returning false on the last line.

Reply:
Good call, might as well simplify as much as we can :D